x-pack/plugins/ml/public/application/util/get_selected_ids_url.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import rison from 'rison-node';
+import { getBasePath } from './dependency_cache';
+
+export enum TAB_IDS {
+  DATA_FRAME_ANALYTICS = 'data_frame_analytics',
+  ANOMALY_DETECTION = 'jobs',
+}
+
+function getSelectedIdsUrl(tabId, settings: any): string {
```

Comment:
sorry to be nit-picky. for type-completeness we should give `settings` its really type here, even if it's basic like `[key: string]:  string | string[]`